src/PureCloud.Client/Apis/WebDeploymentsApi.cs

```diff
@@ -0,0 +1,353 @@
+        }
+
+        var uri = UriHelper.GetUri($"/api/v2/webdeployments/deployments/{deploymentId}", parameters);
+
+        var response = await _httpClient.GetAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<object>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task<CobrowseWebMessagingSession> GetWebdeploymentsDeploymentCobrowseSessionIdAsync(string deploymentId, string sessionId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(deploymentId);
+        ArgumentException.ThrowIfNullOrEmpty(sessionId);
+
+        var uri = UriHelper.GetUri($"/api/v2/webdeployments/deployments/{deploymentId}/cobrowse/{sessionId}", null);
+
+        var response = await _httpClient.GetAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<CobrowseWebMessagingSession>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task<object> GetWebdeploymentsDeploymentConfigurationsAsync(string deploymentId, string type = null, IEnumerable<string> expand = null, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(deploymentId);
+
+        var parameters = new NameValueCollection();
+
+        if (!string.IsNullOrEmpty(type))
+        {
+            parameters.Add("type", type);
+        }
+
+        if (expand != null)
+        {
+            parameters.Add("expand", string.Join(",", expand));
+        }
+
+        var uri = UriHelper.GetUri($"/api/v2/webdeployments/deployments/{deploymentId}/configurations", parameters);
+
+        var response = await _httpClient.GetAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<object>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task<IdentityResolutionConfig> GetWebdeploymentsDeploymentIdentityresolutionAsync(string deploymentId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(deploymentId);
+
+        var uri = UriHelper.GetUri($"/api/v2/webdeployments/deployments/{deploymentId}/identityresolution", null);
+
+        var response = await _httpClient.GetAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<IdentityResolutionConfig>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task<object> GetWebdeploymentsDeploymentsAsync(IEnumerable<string> expand = null, CancellationToken cancellationToken = default)
+    {
+        var parameters = new NameValueCollection();
+
+        if (expand != null)
```

Comment:
Expand is typically an IEnumerable<string> not a single string. Please compare the old code to the new one and determine the correct type